src/pybind/rbd/rbd.pyx

```diff
@@ -898,13 +898,12 @@ class RBD(object):
     def trash_move(self, ioctx, name, delay=0):
         """
         Move an RBD image to the trash.
-
```

Comment:
i did suggest a fix at https://github.com/ceph/ceph/pull/17108#discussion_r134379300, which leaves the newline in. and i think a newline looks better and more consistent with other docstring in this file.